drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -0,0 +1,67 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Hud::Household < Hmis::Hud::Base
+  include ::Hmis::Concerns::HmisArelHelper
+
+  self.table_name = :hmis_households
+  self.primary_key = 'id'
+
+  belongs_to :project, **hmis_relation(:ProjectID, 'Project')
+  has_many :enrollments, **hmis_relation(:HouseholdID, 'Enrollment')
+  has_many :clients, through: :enrollments
+
+  replace_scope :viewable_by, ->(user) do
```

Comment:
I might be missing something, but I think we only need `replace_scope` for some when we're overriding kernel scopes like `system`

Reply:
Oh I think Zach misinterpreted my comment from https://github.com/greenriver/hmis-warehouse/pull/3045#discussion_r1191349740 I meant to suggest deleting this and including the EnrollmentRelated concern (which defines the same thing, using replace_scope). As long as it doesn't hurt to replace a non-existent scope, which I can't imagine it would..

Reply:
Oh ok, didn't realize the concern wouldn't work. Then yeah I think what you have here is right.

Reply:
@ZachCMP  I _believe_ we only need the `replace_scope` when we're replacing something rails provides, but I don't think there's any harm, and it may be desirable if it's overriding another `viewable_by`.  Let's leave it for now.  Thanks.